man/fi_mr.3.md

```diff
@@ -244,7 +244,25 @@ The following apply to memory registration.
 
 *FI_MR_ALLOCATED*
 : When set, all registered memory regions must be backed by physical
-  memory pages at the time the registration call is made.
+  memory pages at the time the registration call is made. In addition,
```

Comment:
We need to update FI_MR_ALLOCATED in fi_domain.3.md as well.

Reply:
I think we're okay here.  The descriptions in fi_domain.md aren't intended to be comprehensive, since they reference back to this page for full details.

Reply:
ok